README.md

```diff
@@ -2,6 +2,8 @@
 
 Marcel attempts to choose the most appropriate content type for a given file by looking at the binary data, the filename, and any declared type (perhaps passed as a request header):
 
+The primary interface is the method `Marcel::MimeType.for`. If the first argument is a `Pathname`, Marcel will evaluate the contents of the specified file. Any other type is assumed to be an `IO`, and Marcel will evaluate its contents.
```

Comment:
I don't think this really fits the tone of the documentation. Let's make the following changes instead:

- Reword the starting sentences to this: "Marcel attempts to choose the most appropriate content type for a given file by looking at the binary data, the filename, and any declared type (perhaps passed as a request header). It's used like this:"
- Add an example above the motivation section with something like "Marcel expects either a `Pathname` or `IO` object to read content type data from: `Marcel::MimeType.for Object.new #  => "application/octet-stream"`"
- Document the `Marcel::MimeType.for` method.

Reply:
Thanks for the comments.

:+1: to documenting `Marcel::MimeType.for`.

I'm not sure I understand the value of the second bullet point. As a new user, the question I had that motivated this PR was: _I want to know all of the ways to use this gem so I can select the one that fits my needs_. It turns out that the gem offers exactly one public interface, but that wasn't spelled out explicitly in the README. I instead propose a sentence at the end of the first section doing so.